src/tasks/LauncherTask.py

```
@@ -0,0 +1,498 @@
+        pid = proc_info.get("pid")
+        if not pid:
+            return 0
+
+        matches = []
+
+        def callback(hwnd, _):
+            if not win32gui.IsWindow(hwnd) or not win32gui.IsWindowEnabled(hwnd):
+                return True
+            try:
+                _, window_pid = win32process.GetWindowThreadProcessId(hwnd)
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)